src/services/vuestic-ui/themes.ts

```diff
@@ -86,63 +40,17 @@ export const COLOR_THEMES = [
       info: '#2c82e0',
       danger: '#e42222',
       warning: '#ffd43a',
-      background: '#f6f7f6',
-      white: '#ffffff',
-      dark: '#262824',
-      gray: '#767c88',
-    },
-    components: {
-      VaNavbar: {
-        color: 'dark',
-        textColor: 'white',
-      },
-      VaSidebar: {
-        color: 'dark',
-        textColor: 'white',
-      },
-      VaCard: {
-        color: 'white',
-      },
-      VaCardContent: {
-        textColor: 'dark',
-      },
-      VaCardTitle: {
-        textColor: 'dark',
-      },
     },
   },
   {
     name: THEME_NAMES.ORIGINAL,
     colors: {
       primary: '#6EDC7D',
       secondary: '#A8AEBA',
-      background: '#eeeeee',
       success: '#76E18D',
       info: '#2c82e0',
       danger: '#e34b4a',
       warning: '#ffc200',
-      gray: '#767c88',
-      white: '#ffffff',
-      dark: '#262824',
-    },
-    components: {
-      VaNavbar: {
-        color: '#1E4CBD',
-        textColor: 'primary',
-      },
-      VaSidebar: {
-        color: '#0A2E80',
-        textColor: 'primary',
-      },
-      VaCard: {
-        color: 'white',
-      },
-      VaCardContent: {
-        textColor: 'dark',
-      },
-      VaCardTitle: {
-        textColor: 'dark',
-      },
     },
```

Comment:
We also lost VaSidebar color.
Previously we had `component: { VaSidebar: { color: 'coolBlueColor' } }`. Checkout master as reference.